### PR TITLE
Update CI config to include Android 7 (API 24)

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
android 5 is missing some functional features we need to test

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,60 +1,68 @@
 language: android
-dist: precise
+dist: trusty
```
Comment (imurchie) [inline .travis.yml]: In my experience Android never works on Travis's "trusty" distribution.

**Comment by mykola-mokhnach** [comment]
Now, finally, after several tens of commits, three days of hard work and four squashes we got our own Travis config with poker and housewives. 

Bender

**Comment by mykola-mokhnach** [comment]
@jlipps Can you please check the PR again, as there were quite a lot of changes since the last review?

File: .travis.yml (hunk 1 above)
Comment (mykola-mokhnach) [reply .travis.yml]: it is strange, but it works with `trusty` now

**Pull request merged**